File: src/sdk-reference/js/6/auth/update-my-credentials/snippets/update-my-credentials.js
const credentials = { username: 'foo', password: 'bar' };

try {
  await kuzzle.auth.login('local', credentials);

  await kuzzle.auth.updateMyCredentials('local', { password: 'worakls' });

  console.log('Success');
} catch (error) {
  console.error(error.message);
}

File: src/sdk/java/1/controllers/auth/update-my-credentials/snippets/update-my-credentials.java
try {
    kuzzle.getAuth().login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
    kuzzle.getAuth().updateMyCredentials("local", "{\"username\":\"foo\",\"password\":\"bar\",\"other\":\"value\"}");
    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
